Update Game Instructions (#2)
cowirihy/reviewing-a-pull-request: README.md

You can play the game at: https://githubschool.github.io/github-games/

### Instructions for playing the game

1. Press the space bar to begin.
2. Use the up and down arrow keys to rotate the shape.
3. Use the left and right arrow keys to position the shape.

> 4. The goal is to create complete rows with no empty spaces.

Reviewer: Why?
```suggestion
4. The objective should be to create complete rows with no empty spaces.
```
